### ISSUE TITLE
Reporting formatting

### ISSUE THREAD

**Issue by mgifford** [w3c/wai-wcag-em#117]
WCAG-EM is a reporting format, however, it is likely made up of other lists or reports that have been made by other systems.

It would be great if there were a W3C opinion or published standard on automated tool output. 

Something that is missing from most systems is just the ability to have a unique identifier for an error that allows for comparison over time or over various pages.

If there is an error in a template (which might just be a fragment of a page), that will likely show up in multiple pages. I should be able to find some way to just label those issues so that they are easier to identify. 

Likewise, over time, an issue that is fixed may re-emerge. This could happen in a software project where a fix is applied in the code, but an update overwrites it. Alternatively, two teams might be working on the same code, and may have mis-communicated. 

At CivicActions we use multiple automated tools (Siteimprove, Axe & Pa11y), plus manual testing for some of our clients. We also get feedback from users and other teams. All of this needs to get acted on within Jira. We have built a tool to consolidate the output from all of these, and then export it into an OpenACR format so that we can provide annual reporting. We have not yet released this tool, but plan to. 

Having a recommendation about reporting formats would make this a lot easier for everyone. Maybe this might also be an [ACT](https://www.w3.org/WAI/standards-guidelines/act/) thing. Certainly I think ACT should be mentioned.

**Comment by hidde** [comment]
The WCAG-EM Report Tool offers this standardised format.
